From: =?utf-8?q?Jeremy_B=C3=ADcha?= <jeremy.bicha@canonical.com>
Date: Sat, 24 Aug 2024 18:28:28 -0400
Subject: Don't require image-avif

Debian's rust-image is currently built without avif support

Gbp-Dch: Name relax-image-avif.diff
---
 Cargo.toml      | 4 ++--
 src/document.rs | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -340,1 +340,0 @@
-avif = ["image/avif-native"]
--- a/src/document.rs
+++ b/src/document.rs
@@ -612,8 +612,6 @@
         "image/gif" => Ok(image::ImageFormat::Gif),
         "image/webp" => Ok(image::ImageFormat::WebP),
 
-        #[cfg(feature = "avif")]
-        "image/avif" => Ok(image::ImageFormat::Avif),
 
         _ => Err(LoadingError::Other(format!(
             "unsupported image format {content_type}"

